addon/components/cells/base.js

```diff
@@ -21,15 +20,10 @@ const Cell = Component.extend({
   layout,
   tagName: 'td',
   classNames: ['lt-cell'],
-  attributeBindings: ['style'],
   classNameBindings: ['align', 'isSorted', 'column.cellClassNames'],
 
   isSorted: computed.readOnly('column.sorted'),
 
-  style: computed('column.width', function() {
```

Comment:
I would argue no. Currently, these `style` attributes don't really have an effect anyway, as the `table-layout: fixed` makes all but the first row ignore custom widths.

As long as the user doesn't overwrite the [`table-layout: fixed` rule](https://github.com/offirgolan/ember-light-table/blob/b0db5b15d96e1b7bfd8f96a3001bfac6338ec0b3/addon/styles/addon.css#L14), nothing should change.

The only case where this could matter is, when the user actively decides to change the [`display`](https://developer.mozilla.org/en-US/docs/Web/CSS/display?v=control) from `table` to something like `flex`.